src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterApplicationServiceImplTest.java

```diff
@@ -256,4 +261,90 @@ void givenInvalidInstitutionId_whenCreatingAdminUserRegisterApplication_thenThro
+                .build();
+
+        // When
+        Mockito.when(adminUserRegisterApplicationRepository.findById(mockId))
+                .thenReturn(Optional.of(mockEntity));
+
+        // Then
+        adminUserRegisterApplicationService.rejectRegistrationApplication(mockId, mockRequest);
+
+        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
+                .findById(mockId);
+    }
+
+    @Test
+    void givenAdminUserRegisterApplicationIdAndAdminUserRegisterApplicationRejectRequest_whenAdminUserRegisterApplicationNotFound_thenThrowAysAdminUserRegisterApplicationNotExistByIdAndStatusException() {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+        AdminUserRegisterApplicationRejectRequest mockRequest = new AdminUserRegisterApplicationRejectRequestBuilder()
+                .withValidFields()
+                .build();
+
+        // When
+        Mockito.when(adminUserRegisterApplicationRepository.findById(mockId))
+                .thenReturn(Optional.empty());
+
+        // Then
+        Assertions.assertThrows(
+                AysAdminUserRegisterApplicationNotExistByIdAndStatusException.class,
+                () -> adminUserRegisterApplicationService.rejectRegistrationApplication(mockId, mockRequest)
+        );
+
+        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
+                .findById(mockId);
+    }
+
+    @Test
+    void givenAdminUserRegisterApplicationIdAndAdminUserRegisterApplicationRejectRequest_whenAdminUserRegisterApplicationIsNotCompleted_thenThrowAysAdminUserRegisterApplicationNotExistByIdAndStatusException() {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+        AdminUserEntity mockAdminUserEntity = new AdminUserEntityBuilder()
+                .withValidFields()
+                .withStatus(AdminUserStatus.NOT_VERIFIED)
+                .build();
+        AdminUserRegisterApplicationEntity mockEntity = new AdminUserRegisterApplicationEntityBuilder()
+                .withValidFields()
+                .withAdminUser(mockAdminUserEntity)
+                .withAdminUserId(mockAdminUserEntity.getId())
+                .withStatus(AdminUserRegisterApplicationStatus.WAITING)
+                .build();
+        AdminUserRegisterApplicationRejectRequest mockRequest = new AdminUserRegisterApplicationRejectRequestBuilder()
+                .withValidFields()
+                .build();
+
+        // When
+        Mockito.when(adminUserRegisterApplicationRepository.findById(mockId))
+                .thenReturn(Optional.of(mockEntity));
+
+        // Then
+        Assertions.assertThrows(
+                AysAdminUserRegisterApplicationNotExistByIdAndStatusException.class,
+                () -> adminUserRegisterApplicationService.rejectRegistrationApplication(mockId, mockRequest)
+        );
+
+        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion

        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Eklendi